### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by i522006**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,9 @@
 
 # SAP Business Technology Platform (SAP BTP) Service Operator for Kubernetes
 
-
-With the SAP BTP service operator, you can provision and consume SAP BTP services in your Kubernetes cluster in a Kubernetes-native way. The SAP BTP service operator is based on the Kubernetes operator pattern to consume SAP BTP services from within the cluster using Kubernetes native tools.
+With the SAP BTP service operator, you can consume [SAP BTP services](https://platformx-d8bd51250.dispatcher.us2.hana.ondemand.com/protected/index.html#/viewServices?) from your Kubernetes cluster using Kubernetes-native tools. 
+SAP BTP service operator allows you to provision and manage service instances and service bindings of SAP BTP services so that your Kubernetes-native applications can access and use needed services from the cluster.  
+The SAP BTP service operator is based on the [CustomResourceDefinitions API](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
```
Comment (eyalnathan) [inline README.md:11]: It is not correct to say that it is based on the CRD API, it is based on the operator pattern - https://kubernetes.io/docs/concepts/extend-kubernetes/operator/

File: README.md [hunk 2]
```diff
@@ -148,82 +155,83 @@ This feature is still under development, review, and testing.
...
-| servicePlanName`*` | `string`   |  The plan to use for the service instance |
-| servicePlanID   |  `string`   |  The plan ID in case service offering and plan name are ambiguous |
-| externalName       | `string`   |  The name for the service instance in SAP BTP, defaults to the binding `metadata.name` if not specified |
-| parameters       |  `[]object`  |  Provisioning parameters for the instance, check the documentation of the specific service you are using for details |
+| serviceOfferingName`*` | `string`  | The name of the SAP BTP service offering. |
+| servicePlanName`*` | `string`   |  The plan to use for the service instance. |
+| servicePlanID   |  `string`   |  The plan ID in case service offering and plan name are ambiguous. |
+| externalName       | `string`   |  The name for the service instance in SAP BTP, defaults to the instance `metadata.name` if not specified. |
+| parameters       |  `[]object`  |  Some services support the provisioning of additional configuration parameters during the instance creation. For the list of supported                                  parameters, check the documentation of the particular service offering.|
```
Comment (eyalnathan) [inline README.md:172]: can you remove the whitespaces? same for the other descriptions in the tables below

File: README.md [hunk 3]
```diff
@@ -148,82 +155,83 @@ This feature is still under development, review, and testing.
...
 
 ### Service Binding 
 #### Spec
-| Parameter             | Type       | Comments                                                                                                   |
+| Parameter             | Type       | Description                                                                                                   |
 |:-----------------|:---------|:-----------------------------------------------------------------------------------------------------------|
-| serviceInstanceName`*`   | `string`   | The Kubernetes name of the service instance to bind, should be in the namespace of the binding |
-| externalName       | `string`   |  The name for the service binding in SAP Cloud Service Management service, defaults to the binding `metadata.name` if not specified |
+| serviceInstanceName`*`   | `string`   | The Kubernetes name of the service instance to bind, should be in the namespace of the binding. |
+| externalName       | `string`   |  The name for the service binding,, defaults to the binding `metadata.name` if not specified |
```
Comment (eyalnathan) [inline README.md:189]: why did you remove the "in SAP Cloud Service Management? for the service instances it says "The name for the service instance in SAP BTP, defaults to the instance `metadata.name` if not specified.". 
I think it is important, as the binding already has a name in the k8s cluster (metadata.name), this name is what's going to show up in BTP (cockpit etc.).

File: README.md [hunk 4]
```diff
@@ -148,82 +155,83 @@ This feature is still under development, review, and testing.
...
-| operationType   |  `string`   |  The operation type (CREATE/UPDATE/DELETE) for ongoing operation |
-| conditions       | `[]condition`   |  An array of conditions describing the status of the service instance. <br>The possible condition types are:<br>- `Ready`: set to `true`  if the instance is ready and usable<br>- `Failed`: set to `true` when an operation on the service instance fails, in this case the error details are available in the condition message.
+| instanceID   | `string`   | The service instance ID in SAP Cloud Service Management service. |
+| operationURL | `string`   |  The URL of the current operation performed on the service instance. |
+| operationType   |  `string`   |  The type of the current operation. Valid values are CREATE, UPDATE, or DELETE.  |
+| conditions       | `[]condition`   |  An array of conditions describing the status of the service instance. <br>The possible condition types are:<br>- `Ready`: set to `true`  if the instance is ready and usable<br>- `Failed`: set to `true` when an operation on the service instance fails. In this case, the details about the error are available in the        condition message.
```
Comment (eyalnathan) [inline README.md:180]: Valid values are usually used for input, as this cannot be set by users i am not sure it is the right way to describe it (same for bindings)

File: README.md (hunk 3 above)
Comment (avilupu) [reply README.md:189]: @eyalnathan do you think " SAP Cloud Service Management" is more accurate than "in SAP BTP"?
The consumer creates instance/binding in K8S, and then sees it in BTP (cockpit etc.), not in "SAP Cloud Service Management".

File: README.md (hunk 1 above)
Comment (i522006) [reply README.md:11]: Changed

File: README.md (hunk 4 above)
Comment (i522006) [reply README.md:180]: Changed to "possible"

File: README.md (hunk 3 above)
Comment (i522006) [reply README.md:189]: Restored

**Review by eyalnathan — APPROVED** [review]
